loadtesting/proxy/tests/scheduled_tx.py

```diff
@@ -160,62 +163,69 @@ class ScheduledTxsIndependentTasksSet(BaseScheduledTxTaskSet):
+                        call_data[i],
+                        child_transaction="0xFFFF",
+                    )
                 )
+            estimate_result = self.web3_client_sol.estimate_scheduled(
+                self.neon_account.solana_account.pubkey(), trx_estimate_obj_list
             )
-        estimate_result = self.web3_client_sol.estimate_scheduled(
-            self.neon_account.solana_account.pubkey(), trx_estimate_obj_list
-        )
 
-        gas_list_new = []
-        for i in estimate_result["gasList"]:
-            new_value = 10 * int(i, 16)
-            gas_list_new.append(hex(new_value))
+            gas_list_new = []
+            for i in estimate_result["gasList"]:
+                new_value = 10 * int(i, 16)
+                gas_list_new.append(hex(new_value))
 
-        estimate_result["gasList"] = gas_list_new
+            estimate_result["gasList"] = gas_list_new
 
-        trxs = []
-        for i in range(trx_count):
-            trxs.append(ScheduledTransaction.from_estimate_result(i, trx_estimate_obj_list[i], estimate_result))
+            trxs = []
+            for i in range(trx_count):
+                trxs.append(ScheduledTransaction.from_estimate_result(i, trx_estimate_obj_list[i], estimate_result))
 
-        tree_acc_data = CreateTreeAccMultipleData(
-            nonce=estimate_result["nonce"],
-            max_fee_per_gas=estimate_result["maxFeePerGas"],
-            max_priority_fee_per_gas=estimate_result["maxPriorityFeePerGas"],
-        )
-        tree_acc_data.add_trx(trxs[0], 0xFFFF, 0)
-        tree_acc_data.add_trx(trxs[1], 0xFFFF, 0)
-        tree_acc_data.add_trx(trxs[2], 0xFFFF, 0)
-        tree_acc_data.add_trx(trxs[3], 0xFFFF, 0)
+            tree_acc_data = CreateTreeAccMultipleData(
+                nonce=estimate_result["nonce"],
+                max_fee_per_gas=estimate_result["maxFeePerGas"],
+                max_priority_fee_per_gas=estimate_result["maxPriorityFeePerGas"],
+            )
+            tree_acc_data.add_trx(trxs[0], 0xFFFF, 0)
+            tree_acc_data.add_trx(trxs[1], 0xFFFF, 0)
+            tree_acc_data.add_trx(trxs[2], 0xFFFF, 0)
+            tree_acc_data.add_trx(trxs[3], 0xFFFF, 0)
 
-        self.evm_loader.create_tree_account_multiple(
-            self.neon_account, self.treasury_pool, tree_acc_data.data, wSOL["address_spl"]
-        )
-        self.web3_client_sol.send_all_scheduled_transactions(trxs)
-        for trx in trxs:
-            check_trx_is_success(self.web3_client_sol, self.evm_loader, trx.hash().hex(), timeout=240)
+            self.evm_loader.create_tree_account_multiple(
+                self.neon_account, self.treasury_pool, tree_acc_data.data, wSOL["address_spl"]
+            )
+            self.web3_client_sol.send_all_scheduled_transactions(trxs)
+            for trx in trxs:
+                check_trx_is_success(self.web3_client_sol, self.evm_loader, trx.hash().hex(), timeout=240)
+        except BaseException:
```

Comment:
what about to move it to separate methods? to base class for example

Reply:
removed